ivy/functional/frontends/torch/creation_ops.py

```diff
@@ -297,3 +301,32 @@ def zeros_like(
 ):
     ret = ivy.zeros_like(input, dtype=dtype, device=device)
     return ret
+
+
+@with_supported_dtypes({"2.0.1 and below": ("float32", "float64")}, "torch")
+@to_ivy_arrays_and_back
+@handle_out_argument
```

Comment:
no need to use handle_out_argument here, this wrapper is for the backends only.
You should use the `out` argument in the ivy functions to inplace update the `out` array.